TSC-Charter.md

```diff
@@ -17,60 +17,48 @@ In the case of the Node.js project, it is delegated to the Node.js
 Technical Steering Committee (“TSC”). OpenJS Foundation’s business
 leadership is the Board of Directors (the “Board”).
 
-This Technical Steering Committee Charter reflects a carefully
-constructed balanced role for the TSC and the CPC in the governance of
-the OpenJS Foundation. The charter amendment process is for the TSC to
-propose changes using simple majority of the full TSC, the proposed
-changes being subject to review and approval by the CPC. The CPC may
-additionally make amendments to the TSC charter at any time, though the
-CPC will not interfere with day-to-day discussions, votes or meetings
-of the TSC.
+This charter can only be amended with the approval of the CPC.
 
 ## Section 3. Establishment of the TSC
 
+TSC members can be either _regular_ members or _voting_ members. Regular
+members can attend meetings and participate in TSC discussions, but do not
+vote. Voting members can do everything regular members can do, and also have
+the ability to cast votes when consensus is not reached on an issue.
```

Comment:
eh, I'm really not all that convinced this is worthwhile, to be honest. Yes, I understand we've had some issues here but I think we need to do a better job of encouraging no-longer active TSC members to just not be TSC members anymore. I'm not going to block, however, if the majority of TSC members want this. Just consider my vote to be -0.

Reply:
I believe https://github.com/nodejs/TSC/pull/1350#issuecomment-1464091435 addresses this line of thinking.